composer.json

```diff
@@ -1,47 +1,51 @@
 {
-  "name": "shopgate/cart-integration-magento2-base",
-  "description": "Shopgate base module for Magento 2, handles communication with merchant API and routing calls",
-  "license": "Apache-2.0",
-  "type": "magento2-module",
-  "version": "2.9.33-alpha1",
-  "authors": [
-    {
-      "name": "Konstantin Kiritsenko"
-    },
-    {
-      "name": "André Kraus"
+    "name": "shopgate/cart-integration-magento2-base",
+    "description": "Shopgate base module for Magento 2, handles communication with merchant API and routing calls",
+    "license": "Apache-2.0",
+    "type": "magento2-module",
+    "version": "2.9.33-alpha1",
+    "authors": [
+        {
+            "name": "Konstantin Kiritsenko"
+        },
+        {
+            "name": "André Kraus"
+        },
+        {
+            "name": "Stephan Recknagel"
+        },
+        {
+            "name": "Alexander Wesselburg"
+        },
+        {
+            "name": "Rainer Skistims"
+        }
+    ],
+    "scripts": {
+        "tests:unit": "../../vendor/bin/phpunit -c phpunit_unit.xml",
+        "tests:integration": "../../vendor/bin/phpunit -c phpunit_integration.xml"
     },
-    {
-      "name": "Stephan Recknagel"
+    "require": {
+        "php": ">=7.1",
+        "shopgate/cart-integration-sdk": "2.9.91",
+        "magento/module-grouped-product": "^100.0",
+        "magento/module-bundle": ">=100.0 <102.0",
+        "magento/module-configurable-product": "^100.0",
+        "laminas/laminas-serializer": "^2.7.2"
     },
-    {
-      "name": "Alexander Wesselburg"
+    "autoload": {
+        "files": [
+            "src/registration.php"
+        ],
+        "psr-4": {
+            "Shopgate\\Base\\": "src/",
+            "Shopgate\\Base\\Tests\\": "tests/"
+        }
     },
-    {
-      "name": "Rainer Skistims"
-    }
-  ],
-  "require": {
-    "php": ">=7.1",
-    "shopgate/cart-integration-sdk": "2.9.91",
-    "magento/module-grouped-product": "^100.0",
-    "magento/module-bundle": ">=100.0 <102.0",
-    "magento/module-configurable-product": "^100.0",
-    "laminas/laminas-serializer": "^2.7.2"
-  },
-  "autoload": {
-    "files": [
-      "src/registration.php"
-    ],
-    "psr-4": {
-      "Shopgate\\Base\\": "src/",
-      "Shopgate\\Base\\Tests\\": "tests/"
-    }
-  },
-  "repositories": [
-    {
-      "type": "composer",
-      "url": "https://repo.magento.com/"
-    }
-  ]
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://repo.magento.com/"
+        }
+    ]
 }
```

Comment:
Indentation should be changed back to 2 spaces. 4 is inconsistent with other modules and makes the diff a bit messed up. Probably was changed accidentally anyway I guess.

Reply:
yup, it seems like it. My IDE has a mind of its own, today it formatted just fine :)